.github/workflows/lint_tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Lint Tests
+
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
Does this run after a merge to main?

Reply:
Hard to tell but just to be sure, I'm adding a main branch check